config/config_base.cmake

```diff
@@ -108,6 +108,9 @@ set(PS_ENCRYPTION                       ON          CACHE BOOL      "Enable encr
 set(PS_CRYPTO_AEAD_ALG                  PSA_ALG_GCM CACHE STRING    "The AEAD algorithm to use for authenticated encryption in Protected Storage")
```

Comment:
Please cleanup the git history.

Parts of the second commit just undoes what happens in the first commit.

I don't see the point of that.

Also, there is no description in the second commit.

Git commits should contain why changes are being made.
What is happening should be self-evident from the code/comments.

Reply:
Yeah that is the plan after the review only count the first commit message